config/default.go

```diff
@@ -196,6 +196,22 @@ Specifies the value of ` + "`" + `User-Agent` + "`" + ` header field used for an
 		label:    "true|false",
 		description: `
 Specifies whether a connection to a worker should be reused.  This overrides [the default keep-alive setting](#env-keep-alive).
+`,
+	},
+	"dispatch_max_conns_per_host": {
+		category:     "common",
+		defaultValue: "10",
+		label:        "<number>",
+		description: `
+Specifies maximum idle connections to keep per-host.
+`,
+	},
+	"dispatch_idle_conn_timeout": {
+		category:     "common",
+		defaultValue: "0",
+		label:        "<seconds>",
+		description: `
+Specifies the maximum time of an idle (keep-alive) connections. If zero, an idle connections will not be closed. 
```

Comment:
[nit]

`s/the maximum time/the maximum amount of time/`

`s/an idle (keep-alive) connections/an idle (keep-alive) connection will remain idle before closing itself/`
or `s/an idle (keep-alive) connections/idle (keep-alive) connections will remain idle before closing themselves/`

`s/an idle connections/an idle connection/`
or `s/an idle connections/idle connections/`